src/__tests__/lollipop-signature-input.test.ts

```diff
@@ -0,0 +1,16 @@
+import { LollipopSignatureInput } from "../generated/definitions/internal/LollipopSignatureInput";
+import * as E from "fp-ts/Either";
+
+const aValidSignatureInput = `sig1=("@method" "@authority" "@path" "content-digest" "content-type" "content-length" "x-pagopa-original-url" "x-pagopa-original-method");created=1618884475;keyid="test-key-ecc-p256", sig2=("@method" "@authority" "@path" "content-digest" "content-type" "content-length" "forwarded");created=1618884480;keyid="test-key-rsa";alg="rsa-v1_5-sha256";expires=1618884540`;
+const anInvalidSignatureInput = `anInvalidStringInput`;
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches and the blocking or non-blocking decorations to further communicate your intent.  -->
**suggestion:** ‏Given the regex tend to be hard to read, I'd provide more examples in tests to serve as documentation on allowed and unallowed signatures. And of course it'll result in a more tested implementation.

Reply:
Thanks, I've tried to improve the unit test in here [42094ec](https://github.com/pagopa/io-functions-fast-login/pull/28/commits/42094eccae6ecb97c4704284d7bf5ac47a8df18f) adding some more descriptive information about this change.